src/python/pants/help/help_formatter.py

```diff
@@ -53,6 +53,13 @@ def add_option(ohis, *, category=None):
             add_option(oshi.advanced, category="advanced")
         if self._show_deprecated:
             add_option(oshi.deprecated, category="deprecated")
+        if oshi.advanced and not self._show_advanced:
+            lines.append(
+                self.maybe_green(
+                    f"There are advanced options which you can list by running "
+                    f"`./pants help-advanced {oshi.scope}`"
```

Comment:
Oh, actually, the `./pants` string should no longer be hard coded.  See:

https://github.com/pantsbuild/pants/pull/14406
https://github.com/pantsbuild/pants/pull/14414

Reply:
Ah thank you, rebased and fixed!

Reply:
Fixed!